pygame/image.py

```diff
@@ -49,13 +51,14 @@ def save(surface, filename):
     fn_normalized = filename.lower()
     result = 0
     # TODO: prep/unprep surface
-    if fn_normalized.endswith('bmp'):
+    if fn_normalized.endswith(as_bytes('bmp')):
```

Comment:
That was my first approach. But then I saw that we had `as_bytes` for this. I'm happy to remove `as_bytes` entirely.